### ISSUE TITLE
Parameters are not generated?

### ISSUE THREAD

**Issue by kaasdude** [kminami/apib2swagger#12]
Hello,

I am trying your nice project.

Got this example:
```
##  Login [/users/login]
Authenticate an user.

+ Model

      + Headers

            Content-Type: application/json
            X-johndoe-token: 1234567890

      + Body

            {
                "username": "johndoe@hotmail.com",
                "password": "123456",
            }

      + Schema

            <!-- include(/docs/schemas/users/login.js) -->

### POST

+ Request

     Login request


+ Response 200 (application/json)

      + Body

            <!-- include(/docs/bodies/users/login.js) -->

      + Schema

            <!-- include(/docs/schemas/users/loginResponse.js) -->
```

What I would expect, is this will generate something like:

```
"/users/login": {
            "post": {
                "responses": {
                    "200": {
                        "description": "OK",
                        "headers": {},
                        "examples": {}
                    }
                },
                "summary": "",
                "description": "Login request",
                "tags": [
                    "Users"
                ],
                "parameters": [
                    {
                        "name": "user",
                        "in": "body",
                        "description": "user to add to the system",
                        "required": true,
                        "schema": {
                            "type": "object",
                            "required": [
                                "username",
                                "password"
                            ],
                            "properties": {
                                "username": {
                                    "type": "string"
                                },
                                "password": {
                                    "type": "string"
                                }
                            }
                        }
                    }
                ]
            }
        },
```

But the converter generates this (without the `parameters` and no `headers` filled in):

```
"/users/login": {
            "post": {
                "responses": {},
                "summary": "",
                "description": "Login request",
                "tags": [
                    "Users"
                ],
                "parameters": []
            }
        },
```

What am I doing wrong?

**Comment by kminami** [comment]
Thank you trying!

First, the Model section should be referred at the Request section.
see this: https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#resource-model-section
e.g. `s/Login request/[Login][]/g`

Second, `<!-- include(path) -->` syntax doesn't seem to be defined at API Blueprint Spec (see above link).
Do you know any tools or extension spec supporting that syntax?

**Comment by kaasdude** [comment]
Thanks for you help!

Let me rephrase the issue:
I made a more stripped version of the blueprint docs and converted to swagger again.

Still missing the parameters stating the body to send.


Blueprint docs look like this:
```
FORMAT: 1A
HOST: http://localhost:1337/v2

# API

Welcome :)

This is the blueprint documentation.


## Login [/users/login]
Authenticate an user.

### Login [POST]

+ Request (application/json)

      {
        "username": "johndoe@hotmail.com",
        "password": "123456"
      }

+ Response 200 (application/json)

      + Body

            {
                user: {
                    data: {
                        email: 'johndoe@hotmail.com',
                        firstName: 'Test',
                        suffix: 'van den',
                        lastName: 'Tester',
                        gender: 'Male'
                    }
                }
            }

```

Is generated to this:
```
{
    "swagger": "2.0",
    "info": {
        "title": "API",
        "version": "",
        "description": "Welcome :)\n\nThis is the blueprint documentation.\n\n\n"
    },
    "host": "localhost:1337",
    "basePath": "/v2",
    "schemes": [
        "http"
    ],
    "paths": {
        "/users/login": {
            "post": {
                "responses": {
                    "200": {
                        "description": "OK",
                        "headers": {},
                        "examples": {}
                    }
                },
                "summary": "Login",
                "description": "",
                "tags": [],
                "parameters": []
            }
        }
    },
    "definitions": {
        "Login": {}
    }
}
```

Expecting / hoping for:
```
{
    "swagger": "2.0",
    "info": {
        "title": "API",
        "version": "",
        "description": "Welcome :)\n\nThis is the blueprint documentation.\n\n\n"
    },
    "host": "localhost:1337",
    "basePath": "/v2",
    "schemes": [
        "http"
    ],
    "paths": {
        "/users/login": {
            "post": {
                "responses": {
                    "200": {
                        "description": "OK",
                        "headers": {},
                        "examples": {}
                    }
                },
                "summary": "Login",
                "description": "",
                "tags": [],
                "parameters": [
                    {
                        "name": "user",
                        "in": "body",
                        "description": "user to add to the system",
                        "required": true,
                        "schema": {
                            "type": "object",
                            "required": [
                                "username",
                                "password"
                            ],
                            "properties": {
                                "username": {
                                    "type": "string"
                                },
                                "password": {
                                    "type": "string"
                                }
                            }
                        }
                    }
                ]
            }
        }
    },
    "definitions": {
        "Login": {}
    }
}
```



I'm sure I made a boo boo somewhere... :-S, not sure..
